services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/utils/GoalStateManagerTest.java

```diff
@@ -4602,6 +4602,343 @@ public void scenarioResourceRequestWithUpdatedAllfields()
             + "}";
     check(input,output);
     }
+
+    @Test
+    public void dhcpTest()
+    {
+    String input =
```

Comment:
Good to have an extra test case. 

One side suggestion: The GoalStateManagerTest.java file becomes a huge file as more test cases are being added. Could you plan to split the payload of each test case with separate conf file, and maintain the main test file in a relatively short manner? 

For example, 
```
public void dhcpTest()
{
	String input = DHCPTestCase.INPUT;
	String output = DHCPTestCase.OUTPUT;
	check(input, output);
}
```

Reply:
A tracking issue #360 has been created.